.palantir/revapi.yml

```diff
@@ -1253,6 +1253,137 @@ acceptedBreaks:
       old: "method void org.apache.iceberg.data.parquet.BaseParquetWriter<T>::<init>()"
       new: "method void org.apache.iceberg.data.parquet.BaseParquetWriter<T>::<init>()"
       justification: "Changing deprecated code"
+  "1.9.0":
```

Comment:
@manuzhang can you rebase please and redo the changes for this file again?

Reply:
Done